cvmfs/cvmfs_config

```diff
@@ -254,13 +257,14 @@ cvmfs_chksetup() {
     fi
   fi
 
+  AUTOMASTER=$(find /etc -name "auto.master" 2>/dev/null | head -1)
```

Comment:
This can potentially take a long time.  I assume that auto.master is not directly under /etc on archlinux, right?  Can we just test for /etc/auto.master and then for the alternative location?